### PR TITLE
DM-26641: Problem with log plots and problem with bad amps in ptc.py

### PR DESCRIPTION
**Pull request by plazas**

### CODE REVIEW

File: python/lsst/cp/pipe/plotPtc.py [hunk 1]
```diff
@@ -810,50 +817,63 @@ def _plotStandardPtc(self, dataset, ptcFitType, pdfPages):
...
+                       s=markerSize)
+            a3.text(0.2, 0.65, stringLegend, transform=a3.transAxes, fontsize=legendFontSize)
+            a3.set_title(amp, fontsize=titleFontSize)
+            a3.set_xlim([minMeanVecOriginal - 0.2*deltaXlim, maxMeanVecOriginal + 0.2*deltaXlim])
 
         f.suptitle("PTC \n Fit: " + stringTitle, fontsize=supTitleFontSize)
         pdfPages.savefig(f)
         f2.suptitle("PTC (log-log)", fontsize=supTitleFontSize)
         pdfPages.savefig(f2)
+        f3.suptitle("Var/$mu$", fontsize=supTitleFontSize)
```
Comment (czwa) [inline python/lsst/cp/pipe/plotPtc.py:875]: This needs a backslash I believe to properly format the μ.

File: python/lsst/cp/pipe/plotPtc.py [hunk 2]
```diff
@@ -810,50 +817,63 @@ def _plotStandardPtc(self, dataset, ptcFitType, pdfPages):
...
+
+            # Same, but in log-scale
+            a2.plot(meanVecFit, ptcFunc(pars, meanVecFit), color='red')
+            a2.scatter(meanVecFinal, varVecFinal, c='blue', marker='o', s=markerSize)
+            a2.scatter(meanVecOutliers, varVecOutliers, c='magenta', marker='s', s=markerSize)
+            a2.text(0.03, 0.7, stringLegend, transform=a2.transAxes, fontsize=legendFontSize)
+            a2.set_title(amp, fontsize=titleFontSize)
+            a2.set_xlim([minMeanVecOriginal, maxMeanVecOriginal])
+
+            # Var/mu vs mu
```
Comment (czwa) [inline python/lsst/cp/pipe/plotPtc.py:862]: Should the docstring be updated to note the addition of this plot?

File: python/lsst/cp/pipe/plotPtc.py [hunk 3]
```diff
@@ -907,27 +932,23 @@ def _plotLinearizer(self, dataset, linearizer, pdfPages):
...
-                a2.plot(meanVecFinal, fracLinRes, c='g')
-                a2.set_title(f"{amp}", fontsize=titleFontSize)
-            else:
-                a.set_title(f"{amp} (BAD)", fontsize=titleFontSize)
-                a2.set_title(f"{amp} (BAD)", fontsize=titleFontSize)
+            pars, parsErr = linearizer.fitParams[amp], linearizer.fitParamsErr[amp]
+            k0, k0Error = pars[0], parsErr[0]
+            k1, k1Error = pars[1], parsErr[1]
+            k2, k2Error = pars[2], parsErr[2]
+            linRedChi2 = linearizer.linearityFitReducedChiSquared[amp]
```
Comment (czwa) [inline python/lsst/cp/pipe/plotPtc.py:939]: `linearizer` does not have this property anymore; it's now `fitChiSq`.

File: python/lsst/cp/pipe/plotPtc.py [hunk 4]
```diff
@@ -907,27 +932,23 @@ def _plotLinearizer(self, dataset, linearizer, pdfPages):
...
+            stringLegend = (f"k0: {k0:.4}+/-{k0Error:.2e} DN\n k1: {k1:.4}+/-{k1Error:.2e} DN/t"
```
Comment (czwa) [inline python/lsst/cp/pipe/plotPtc.py:940]: I don't see an example of this attached to the jira ticket, but doesn't this introduce spaces before the k1 and k2 lines?

**Review by czwa — APPROVED** [review]
Minor comments, other than the change that's needed due to DM-24703 merging yesterday.

File: python/lsst/cp/pipe/plotPtc.py (hunk 1 above)
Comment (timj) [reply python/lsst/cp/pipe/plotPtc.py:875]: Can't you just use the unicode character directly?

Comment (czwa) [reply python/lsst/cp/pipe/plotPtc.py:875]: The rest of the code uses LaTeX style, so I think it's better to be consistent.

File: python/lsst/cp/pipe/plotPtc.py (hunk 4 above)
Comment (plazas) [reply python/lsst/cp/pipe/plotPtc.py:940]: For this ticket I did not produce a linearizer; but I can check.

**Pull request merged**